turborepo-tests/integration/tests/dry_json/single_package_pnpm.t

```diff
@@ -0,0 +1,92 @@
+Setup
+  $ . ${TESTDIR}/../../../helpers/setup.sh
+  $ . ${TESTDIR}/../_helpers/setup_monorepo.sh $(pwd) single_package pnpm@8.0.0
+
+  $ ${TURBO} run build --dry=json
```

Comment:
I would just assert the exit code not the whole output. when dry run output changes its annoying to go update a bunch of unrelated tests